assets/pages/monthly-contributions/monthlyContributions.jsx

```diff
@@ -37,10 +37,14 @@ pageStartup.start();
 
 // ----- Redux Store ----- //
 
-const store = createStore(reducer, {
-  intCmp: getQueryParameter('INTCMP'),
-}, applyMiddleware(thunkMiddleware));
-
+/* eslint-disable no-underscore-dangle */
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
Yes.  Contributions have something similar in prod: https://github.com/guardian/contributions-frontend/commit/fcb79123c46fcf07073865854c168b928e2a55b7